Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StructArrayTest.verified.txt

```diff
@@ -0,0 +1,2 @@
+﻿System.Void <Module>::test()
+  IL_0000: ret
```

Comment:
This one is also not very useful. I assume we'd need to upgrade these tests to handle the global variables correctly.